src/etc/inc/dyndns.class

```diff
@@ -338,11 +338,15 @@
 			$this->_dnsUpdateURL = $dnsUpdateURL;
 			$this->_dnsResultMatch = $dnsResultMatch;
 			$this->_dnsRequestIf = get_failover_interface($dnsRequestIf);
+			if ($dnsMaxCacheAge !== "") {
+				$this->_dnsMaxCacheAgeDays = (int)$dnsMaxCacheAge;
+			} else {
+				$this->_dnsMaxCacheAgeDays = 25;
```

Comment:
This is written with `if..else`, because #4514 will add `switch` here.

@jim-p to your info, I changed this part after your review.